apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -258,6 +262,30 @@ export default function SectionsSetUpContainer({
     );
   };
 
+  const renderChildAccountPolicyNotification = () => {
```

Comment:
I'm leaning toward leaving it as is. Since this Notification banner is only used in this component, and to avoid complications given the tight deadline.